okapi-core/src/main/raml/ModuleUser.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "LaunchDescriptor.json",
+  "title": "LaunchDescriptor",
+  "description": "Specifies the user that should be created for the module",
+  "type": [ "object", "null" ],
+  "additionalProperties" : false,
+  "properties": {
+    "type": {
+      "description": "The type of the module user",
+      "type": ["string"]
+    },
+    "permissions": {
+      "description": "Permission ID (usually module.service.method or similar)",
+      "type": ["array", "null"],
+      "items": {
+        "type": "string",
+        "enum": ["module", "tenant"],
+        "default": "module"
+      }
```

Comment:
Why is this an enum and not the "type" earlier.. I thought the purpose of this array was to list the permissions needed for that user. If we don't, how do we know what the module needs?

Reply:
done